ksqldb-functional-tests/src/test/resources/query-validation-tests/group-by.json

```diff
@@ -362,6 +362,100 @@
         ]
       }
     },
+    {
+      "name": "subscript in group-by and select",
```

Comment:
can we add corresponding tests for `HAVING`?

Reply:
I am a bit weirded out but the behavior of `HAVING`: Why does it return rows for non-matching keys with null value?

Reply:
Because of this: https://github.com/confluentinc/ksql/issues/3558